src/engine/renderer/tr_shader.cpp

```diff
@@ -6194,6 +6181,16 @@ shader_t       *R_FindShaderByName( const char *name )
 	return tr.defaultShader;
 }
 
+static void ClearGlobalShader()
+{
+	shader = {};
+
+	for ( size_t i = 0; i < MAX_SHADER_STAGES; i++ )
```

Comment:
Excellent, this revealed a bug (some other part of the code was using `stages` instead of `stage`, and then always `stage[0]`:

```
tr_shader.cpp:3231:25: error: base operand of ‘->’ has non-pointer type ‘std::array<shaderStage_t, 16>’
 3231 |         else if ( stages->collapseType == collapseType_t::COLLAPSE_REFLECTIONMAP )
```

Reply:
Used `std::array` and `ResetStruct`. 👍️